src/index.html

```diff
@@ -7,8 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <li class="list list__apple">
+          <a class="text is-active" href="#">
+            apple
+          </a>
+        </li>
+
```

Comment:
use below structure are mentioned by Dima
```
<ul> -- list
  <li> -- list item (1)
    <a> -- link
  </li>
  <li> -- list item (1)
    <a> -- link
  </li>
  ...
</ul>
```

Reply:
as I have mentioned add class `nav__list ` to `ul`  tag
add class `nav__item ` to `li`  tag
add class `nav__link` to `a`  tag